src/CoreBundle/Resources/public/javascript/modules/sidebar/sidebar.js

```diff
@@ -31,7 +32,25 @@
 
             this.$baseElement.on("keyup", this.handleKeyEvent.bind(this));
 
+            this.$baseElement.find("nav").on("scroll", function(evt) {
+                console.log("Setting "+ $(this).scrollTop());
+                localStorage.setItem('dummy-pos-test', $(this).scrollTop());
+            });
+
             this.$filterElement.focus();
+
+            if (null !== localStorage.getItem('dummy-pos-test')) {
+                setTimeout(function() {$(".sidebar-nav")[0].scrollTop = localStorage.getItem('dummy-pos-test');}, 1000);
+                //$(".sidebar-nav")[0].scrollTop = localStorage.getItem('dummy-pos-test');
```

Comment:
I don`t get what this dummy-pos-test is used for. Should be explained in the code.

Reply:
Code remove / moved to #522